aleph-client/src/lib.rs

```diff
@@ -217,6 +216,48 @@ pub trait AnyConnectionExt: AnyConnection {
     }
 }
 
+pub trait BalanceTransfer {
```

Comment:
I think we need to come up (I'm okay with not doing it in this PR, yet still I think it's an important thing to consider) with a holistic approach to such a new interface that abstracts `aleph-client` from any library that provides data to it. They are loosely coupled to me now, maybe we should make them resemble pallets and their calls? E.g.

```
trait PalletBalances {
    fn transfer(&self, tx: Self::TransferTx, status: XtStatus)
        -> Result<Option<H256>, Self::Error> { {// some default imp }
     fn force_transfer(&self, tx: Self::TransferTx, status: XtStatus)
        -> Result<Option<H256>, Self::Error> { {// some default imp  }
   ....
}

trait PalletSystem {
   fn fill_block(&self, target_ratio: u32, status: XtStatus) -> Result<(), Self::Error> {// some default imp }
};
```
 
We need to provide some defaults impl to actually make interface segregation possible. Also we can then extend those traits as we go with more pallet calls. Plus of that design is that pallets and their names in substrate should not change very often, as they are API themselves.

Reply:
Let's keep that in mind during `subxt` migration